content/ui-patterns/messaging.mdx

```diff
@@ -7,25 +7,157 @@ import {Box} from '@primer/react'
 
 Primer includes three different messaging components:
 
-- Toasts
-- [Flash alerts](../components/flash)
-- [Popovers](../components/popover) 
+- Announcement
+- [Banner](../components/flash)
+- [Popover](../components/popover)
+- Toast
+- Callout
```

Comment:
I would say that Callout is not a way to notify users, it's about emphasizing content. I would mention it as part of Banner and refer to it, but wouldn't want to confuse to use it for notifications (unless this page continues to be about all things messaging but it's getting a lot).